### PR TITLE
Minor query cleanups

### PR DESCRIPTION
**Pull request by pveentjer**

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/query/impl/CachedQueryEntry.java [hunk 1]
```diff
@@ -110,34 +107,44 @@ public Comparable getAttribute(String attributeName) throws QueryException {
...
-                    targetObject = getKey();
-                }
+            return getTargetKey();
+        } else {
+            return getTargetValue();
+        }
+    }
+
+    private Object getTargetValue() {
+        Object targetObject;
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/query/impl/CachedQueryEntry.java]: You could safe some lines of code and indentation with direct `return` statements. Also the definition order of `getTargetValue()` and `getTargetKey()` methods should be the same as the invocation order.

File: hazelcast/src/main/java/com/hazelcast/query/impl/IndexImpl.java [hunk 2]
```diff
@@ -50,10 +50,10 @@
 
     private final SerializationService ss;
 
-    public IndexImpl(String attribute, boolean ordered, SerializationService ss) {
+    public IndexImpl(String attribute, boolean ordered, SerializationService serializationService) {
         this.attribute = attribute;
         this.ordered = ordered;
-        this.ss = ss;
+        this.ss = serializationService;
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/query/impl/IndexImpl.java]: Forgot to rename the field as well?

File: hazelcast/src/main/java/com/hazelcast/map/impl/query/QueryEntryFactory.java [hunk 3]
```diff
@@ -24,13 +24,14 @@
 
 public final class QueryEntryFactory {
     private final boolean useCache;
+    private final SerializationService serializationService;
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/map/impl/query/QueryEntryFactory.java]: Could you please align the field and constructor parameter order if you touch this PR anyway?

Pleaaaze

**Comment by Donnerbart** [comment]
Minor comments :+1:

**Comment by jerrinot** [comment]
:+1:  thanks!

**Pull request closed without merging**